### DISCUSSION TITLE
将网页端首页推荐替换为APP端首页推荐/添加对[B 站网页端添加 APP 首页推荐]脚本的支持

### DISCUSSION THREAD

**Discussion by Arragon** [the1812/Bilibili-Evolved#2913] [功能建议 / Ideas]
希望能够添加和[ B 站网页端添加 APP 首页推荐 ](https://github.com/indefined/UserScripts/tree/master/bilibiliHome)这个脚本一样的将网页端热门推荐替换为APP个人首页推荐的插件。
或者能够对这个脚本提供兼容支持。
感谢您的辛勤付出。

**Comment by Geeyun-JY3** [comment]
https://github.com/the1812/Bilibili-Evolved#相关推荐
> 相同的功能将不会整合到 Bilibili Evolved, 但会尽可能地适配

现在新版首页该脚本不支持吧：https://github.com/indefined/UserScripts/issues/76
旧版首页不适配了吗？

**Reply by Arragon** [reply]
我之前一直在使用Evolved V1，因为当时两个脚本不兼容所以并没有太关注[B 站网页端添加 APP 首页推荐]，还以为它在新版也有效。今天突然想起来才提了这个建议，没做深入了解。
我刚刚试了一下，旧版脚本依然有效，同时两个脚本在旧版可以兼容。

**Reply by the1812** [reply]
v2 不兼容这个脚本吗?

**Reply by Arragon** [reply]
抱歉我可能表述有问题。V2与这个脚本在旧版主页是可以兼容的。

**Reply by the1812** [reply]
你可以先用着旧版主页, 我这边没有实现 APP 首页推荐的意愿.
